editor/debugger/debug_adapter/debug_adapter_parser.cpp

```diff
@@ -193,7 +207,18 @@ Dictionary DebugAdapterParser::_launch_process(const Dictionary &p_params) const
 
 	String platform_string = args.get("platform", "host");
 	if (platform_string == "host") {
-		EditorRunBar::get_singleton()->play_main_scene();
+		Vector<String> play_args = _extract_play_arguments(args);
+
+		if (args.has("scene")) {
+			const String scene = args["scene"];
+			if (scene == "main") {
+				EditorRunBar::get_singleton()->play_main_scene(false, play_args);
+			} else if (scene == "current") {
+				EditorRunBar::get_singleton()->play_current_scene(false, play_args);
+			} else {
+				EditorRunBar::get_singleton()->play_custom_scene(scene, play_args);
+			}
+		}
```

Comment:
```suggestion
		} else {
			EditorRunBar::get_singleton()->play_main_scene(false, play_args);
		}
```

We should fallback to main scene if the `scene` argument wasn't provided.